validation/management/commands/findorigin.py

```diff
@@ -33,27 +33,27 @@ def handle(self, *args, **options):
         md_words = get_md_words()
         phrases = Phrase.objects.all()
         for phrase in tqdm(phrases.iterator(), total=phrases.count()):
-            if not phrase.origin:
-                normalized = normalize_sro(phrase.transcription)
-                normalized = remove_diacritics(normalized)
-                if normalized in md_words:
-                    phrase.origin = "MD"
-                else:
-                    phrase.origin = "new"
-                phrase.save()
+            normalized = normalize_sro(phrase.transcription)
+            normalized = remove_diacritics(normalized)
```

Comment:
Is it for sure that a term in the Maskwacîs dictionary will be written in normalized SRO, without diacritics?